### PR TITLE
Release 0.15.1 changelog

### PR DESCRIPTION
**Pull request by fjork3**

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -2,6 +2,36 @@
...
+* [MAINTENANCE] Fix Typo in Checkpoint Readme (#4835) (thanks @andyjessen)
+* [MAINTENANCE] Modify conditional expectations readme (#4616) (thanks @andyjessen)
+* [MAINTENANCE] Fix links within datasource new notebook (#4833) (thanks @andyjessen)
+* [MAINTENANCE] Adds missing dependency, which is breaking CLI workflows (#4839)
+* [MAINTENANCE] Update testing and documentation for `oneshot` estimation method (#4852)
+* [MAINTENANCE] Refactor `Datasource` tests that work with `RuntimeDataConnector` by backend.  (#4853)
+* [MAINTENANCE] Update DataAssistant interfaces (#4857)
+* [MAINTENANCE] Improve types returned by DataAssistant interface methods (#4859)
+* [MAINTENANCE] Refactor `DataContext` tests that work with RuntimeDataConnector by backend (#4858)
+* [HACKATHON] [Hackathon PRs in this release](https://github.com/great-expectations/great_expectations/pulls?q=is%3Apr+label%3Ahackathon-2022+is%3Amerged+-updated%3A%3E%3D2022-04-14+-updated%3A%3C%3D2022-04-06)
```
Comment (austiezr) [inline docs/changelog.md:31]: ❤️

File: docs/changelog.md [hunk 2]
```diff
@@ -2,6 +2,36 @@
 title: Changelog
 ---
 
+### 0.15.1
+* [FEATURE] Additional Rule-Based Profiler Parameter/Variable Access Methods (#4814)
+* [FEATURE] add icd10 regex expectation (#4691) (thanks @zachlindsey)
+* [FEATURE] Expect column values to be http method (#4690) (thanks @zachlindsey)
```
Comment (austiezr) [inline docs/changelog.md]: These two are hackathon entries that were mis-labeled; they're listed in the below PRs link as well.

**Review by austiezr — APPROVED** [review]
Small note but not gating 🖤

File: docs/changelog.md (hunk 2 above)
Comment (fjork3) [reply docs/changelog.md]: fixed in PR title and removed from changelog

**Pull request merged**